python/python/res/enkf/enkf_main.py

```diff
@@ -73,27 +74,19 @@ class EnKFMain(BaseCClass):
     _get_res_config = EnkfPrototype("res_config_ref enkf_main_get_res_config(enkf_main)")
 
 
-    def __init__(self, model_config, res_config=None, strict=True, verbose=True):
-        if model_config is not None and not isfile(model_config):
-            raise IOError('No such configuration file "%s".' % model_config)
+    def __init__(self, config, strict=True, verbose=True):
```

Comment:
Exactly, it gives a deprecationwarning if the name of a configuration file is provided. I'll write a comment.